src/CMakeLists.txt

```diff
@@ -465,9 +474,18 @@ elseif(APPLE)
         endif(IMAGER_SIGNING_IDENTITY)
     else()
         # Unsigned application
+        message(STATUS "App will be unsigned")
+        add_custom_command(TARGET ${PROJECT_NAME}
+            POST_BUILD
+            COMMAND "${MACDEPLOYQT_EXECUTABLE}" "${CMAKE_BINARY_DIR}/${PROJECT_NAME}.app" -qmldir="${CMAKE_CURRENT_SOURCE_DIR}" -always-overwrite -no-strip -dmg)
+        # Rename .app file
```

Comment:
If you wanted to do this, you probably wanted to remove -dmg above - no point in creating a dmg containing the original name.